docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1505,23 +1508,230 @@ To properly configure the FHIR server's keystore and truststore files, perform t
+### 5.2.3.1 Configure the server
+Copy the server keystore (`serverKeystore.jks`) and truststore (`serverTruststore.jks`) files to the appropriate directory (`$WLP_HOME/usr/servers/fhir-server/resources/security`). Then configure the `server.xml` file correctly to reference your new keystore and truststore files.
 
-Although the steps required to configure certificate-based authentication for a client application depend on the specific REST API client framework, the client has several main requirements, as summarized in the following list:
+### 5.2.3.2 Configure the client
+The precise steps required to configure certificate-based authentication for a client application depend on the specific REST API client framework, but these are the general rules:
 
 *   If the client is using the FHIR server's HTTPS endpoint, then the client truststore should be configured with the REST API client framework<sup id="a11">[11](#f11)</sup>.
 *   If the client is using basic authentication, then it must send an appropriate Authorization request header containing the username and password information in the HTTP request.
 *   If the client is using client certificate-based Authentication, then the client keystore must be configured with the REST API client framework<sup id="a12">[12](#f12)</sup>.
 *   If the client is using OAuth 2.0 Authentication, then the client keystore must be configured with the REST API client framework. In addition, it must send an appropriate Authorization request header containing the Bearer token in the HTTP request.
 
-### 5.2.5 OAuth 2.0
-In the default configuration, the FHIR server acts as an authorization server as well as a resource server. The FHIR server's conformance statement includes the OAuth-related URLs you will need to get started. The following excerpt from a conformance statement shows sample OAuth-related URLs (token, authorize, and register) as values of the `valueUri` elements.
+## 5.3 OpenID Connect and OAuth 2.0
+The FHIR specification recommends the use of OpenID Connect and OAuth 2.0.
+The IBM FHIR Server supports these via Liberty's OpenID Connect support.
+The following sections are adapted from https://www.ibm.com/support/knowledgecenter/SSD28V_liberty/com.ibm.websphere.wlp.core.doc/ae/twlp_config_oidc_pc_examp_beginner.html and related pages in the WebSphere Liberty knowledge center, but they apply to OpenLiberty
+as well.
+
+### 5.3.1 Configure Liberty as the OpenID Connect Provider
+Liberty can be configured to act as an OpenID Connect Provider via the [openidConnectServer-1.0 feature](https://openliberty.io/docs/ref/feature/#openidConnectServer-1.0.html).
```

Comment:
maybe mention that the server.xml has the feature enabled by default? it's just not configured?

Reply:
```suggestion
Liberty can be configured to act as an OpenID Connect Provider via the [openidConnectServer-1.0 feature](https://openliberty.io/docs/ref/feature/#openidConnectServer-1.0.html). This feature is enabled by default, but must be configured as described in the following sections to take effect.
```